backend/imageverify/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:18-slim
+
+# Install dependencies for canvas
+RUN apt-get update && apt-get install -y 
+    build-essential 
```

Hadolint (2.14.0)

[error] unexpected 'b'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)